src/components/SectionTitle/SectionTitle.tsx

```diff
@@ -17,7 +17,7 @@ export function SectionTitle({ label, title, description, ...layoutProps }: Sect
     <div css={layoutCss} {...layoutProps}>
       <h4 css={labelCss}>{label}</h4>
       <h2 css={titleCss}>{title}</h2>
```

Comment:
나머지 친구들도 같은 맥락으로 이렇게 하는 건 어때요?? title, label이 항상 쓰여서 별 의미는 없지만 통일성!
```suggestion
     {label && <h4 css={labelCss}>{label}</h4>}
     {title && <h2 css={titleCss}>{title}</h2>}
```

Reply:
적용했습니다!